notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -336,7 +336,7 @@ export default {
       }
       setTimeout(() => this.hasManualChildren = false, 100);
       if ( this.note && this.note.breadcrumb && this.note.breadcrumb.length ) {
-        this.note.breadcrumb[0].title = this.getHomeTitle(this.note.breadcrumb[0].title);
+        this.note.breadcrumb[0].title = this.$t('notes.label.noteHome');
```

Comment:
You need to check what was the logic added in the function getHomeTitle() and not remove it